Dockerfile.windows.ci

```diff
@@ -0,0 +1,38 @@
+# Temp Core Image
+FROM microsoft/windowsservercore AS core
+
+ENV RUBY_VERSION 2.2.4
```

Comment:
Why this old version of Ruby?

Reply:
ETOOHARD basket. Maybe another day, I don't think it should make much of a difference.